lib/wallaroo/ent/router_registry/router_registry.pony

```diff
@@ -684,15 +684,9 @@ actor RouterRegistry is InFlightAckRequester
     """
     Start the log rotation and initiate snapshots.
     """
-    let steps: Map[U128, Step] iso = recover steps.create() end
-    for pr in _partition_routers.values() do
-      for (i, v) in pr.local_map().pairs() do
-        steps(i) = v
-      end
-    end
     match _event_log
     | let e: EventLog =>
-      e.rotate_file(consume steps)
```

Comment:
I changed how this works because now when we rotate a file, we need to ensure that all `Resilient` actors are involved (and not only `Step`s). The `event_log` now uses the list of `Resilients` it already has from when they register with it instead of expecting a map of steps to be passed to it.

See https://github.com/WallarooLabs/wallaroo/pull/2140/files/34a78f77efa9328e10824ffb824db60552123e4e#diff-20620bad0b2c64ccae73559b3fbc9736L259 for specific changes.

Reply:
Looks good